engine/src/rgbmatrix.cpp

```diff
@@ -242,6 +242,10 @@ void RGBMatrix::setAlgorithm(RGBAlgorithm *algo)
                     m_properties.take(it.key());
                 }
             }
+
+            QVector<uint> colors = script->rgbMapGetColors();
+            for (int i = 0; i < colors.count(); i++)
+            	m_rgbColors.replace(i, QColor::fromRgb(colors.at(i)));
```

Comment:
i see a tab here. I guess the codestyle of this project implies spaces instead of tabs

Reply:
303438163be901eb27b6b7e80abfb66ba8202109